packages/@uppy/aws-s3-multipart/src/MultipartUploader.js

```diff
@@ -243,14 +243,22 @@ class MultipartUploader {
 
   async #prepareUploadParts (candidates) {
     this.lockedCandidatesForBatch.push(...candidates)
+    let result
 
-    const result = await this.options.prepareUploadParts({
-      key: this.key,
-      uploadId: this.uploadId,
-      partNumbers: candidates.map((index) => index + 1),
-    })
+    try {
+      result = await this.#retryable({
+        attempt: () => this.options.prepareUploadParts({
+          key: this.key,
+          uploadId: this.uploadId,
+          partNumbers: candidates.map((index) => index + 1),
+        }),
+      })
+    } catch (error) {
+      throw new Error(error)
```

Comment:
why wrap the error in another error? I think we can just remove the whole try/catch

Reply:
Ah I misunderstood. I thought we still need to throw like the error below to get it up. But `retryable` already throws the error and it should move up to where it's handled.